### ISSUE TITLE
stellar: send does not work

### ISSUE THREAD

**Issue by prusnak** [trezor/trezor-core#447]
https://www.reddit.com/r/Stellar/comments/abynfg/send_fails_with_stellar_account_viewer/

We have to check this, whether this is correct and whether it affects T1 or not.

**Comment by pankrats** [comment]
Yes, exactly the same issue with T1 also: 

> However, when I get to the screen where I'm confirming the sending value, I see:
> "Pay 0 XLM (native) to : <correct-reciepent-address>"
> Then when I press and hold for the final confirmation, I see the transaction failed message in the Stellar Account Viewer.

**Comment by bosomt** [comment]
I already reported it to Stellar devs here: https://github.com/stellar/go/issues/795
No response. 
@tsusanka told me that he will check it

**Comment by pankrats** [comment]
reported here https://github.com/stellar/go/issues/795

**Comment by tsusanka** [comment]
Will do on Monday

**Comment by tsusanka** [comment]
Summary:

- trezor.io/stellar is buggy. When attempting to sign a transaction Trezor does not display anything so it most likely does not communicate with it properly.
  - @prusnak will try to update trezor.io/stellar to use newer Connect, but it's a bit of pain since stellar uses [interstellar](https://www.npmjs.com/package/interstellar) which is now deprecated.

- Singing using the [Connect Explorer](https://trezor.github.io/connect-explorer/#/stellar-signtx) seems fine. I haven't tried to publish the transaction to the network yet, but the signing goes fine as expected.

- Same goes for [stellar trezor inflation tool](https://kolontsov.github.io/stellar-trezor-inflation/). The signing goes fine and I also successfully broadcasted the transaction to the network.

So in a nutshell, I don't think this is an error on the firmware side. It might be the case for the ChangeTrustOp, which I'll investigate as part of https://github.com/trezor/trezor-core/issues/421.

**Comment by tsusanka** [comment]
What confuses me a little is that I'm not getting the same behaviour as reported by the reddit user and @pankrats. But let's see if we succeed to update the account viewer.

**Comment by tsusanka** [comment]
Seems fixed :tada:. @bosomt could you pls check if it works for both T1 and TT? I've tried with T emulator only

**Comment by reqlez** [comment]
> Seems fixed 🎉. @bosomt could you pls check if it works for both T1 and TT? I've tried with T emulator only

I have a T, instructions to test? if you want I can do that as well. is this now live on trezor.io/stellar ?

**Comment by tsusanka** [comment]
Sure, if you want just give it a go. trezor.io/stellar should be working now

**Comment by reqlez** [comment]
Actually, just tested with T and Chrome ( wihout the bridge software installed, if that matters ). Works great !  May I suggest getting the inflation destination code on the page going as well ;-)

**Comment by tsusanka** [comment]
Apparently, there is still an issue with the account viewer when sending to a different address than yours. I'll investigate

**Comment by reqlez** [comment]
I sent to a different address when I tested...

**Comment by tsusanka** [comment]
There was a problem with addresses that did not have any lumens on it yet. The Payment op was called instead of the CreateAccount op. We already have a fix and it will be deployed soon. I'll close this issue then

**Comment by prusnak** [comment]
Fix deployed. Try trezor.io/stellar whether it works.

**Comment by bosomt** [comment]
consider it fixed.
Transactions are working fine now !

**Comment by tsusanka** [comment]
Confirmed :tada:
